stories/Button.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { storiesOf } from "@storybook/react";
+import { Button } from "../src/renderer/components/Button";
+
+storiesOf("Button", module)
```

Comment:
I wonder if it's better to put stories next to actual component?

Reply:
That's possible too but I don't know if that's better. When I think of CSS, if everything won't be imported by `index.scss`  (but sometimes in the component specific for that file), then it is better.

Location-wise I don't have any feelings. If yes for this, then what about redux functions and tests?

Reply:
lets leave it like that. Not sure I understand thing about css?

Reply:
Nevermind the css but another thing occurred to me. We'll have to keep the same structure in `stories` directory as well as `src` regarding components/containers. I now kinda lean towards the suggested approach xD

Reply:
haha yeah, make sure that build doesnt include .stories.ts in output directory